Build.md

```diff
@@ -16,23 +16,23 @@ At any time, you can pull changes from the upstream and merge them onto your mas
 
 The general idea is to keep your 'master' branch in-sync with the 'upstream/master'.
 
-## Building JBoss Fuse Tooling
-To build _JBoss Fuse Tooling_ requires specific versions of Java (1.6+) and +Maven (3.0+). 
+## Building JBoss Camel Developer Tools
```

Comment:
look at tools.jboss.org. there we already have this name. But I will double check with Aileen.